### PR TITLE
database-monitoring troubleshooting: add connection test section

### PR DESCRIPTION
**Pull request by EmmaOnThursday**
### What does this PR do?
Adds a brief section on checking for DBM endpoint connection issues to public troubleshooting docs. 

### Motivation
These steps have been helpful for us in diagnosing issues, and making them publicly available might encourage customers to take these steps themselves before contacting support. Additionally, having a public link for these steps reduces the need to copy-paste commands around in support tickets. 

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/database_monitoring/troubleshooting.md [hunk 1]
```diff
@@ -28,6 +28,40 @@ Depending on how complex the databases being monitored are, too many DBM hosts o
...
+curl -vvv -X POST "https://dbquery-intake.datadoghq.com/api/v2/databasequery" \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "DD-API-KEY: NONE" \
+-d "[{}]"
+```
+
+The responses should contain `{"status":"error","code":403,"errors":["Forbidden"],...}` if requests were successfully sent and a response was received.
+
+Some common causes of connection failure include proxy setups and firewalls preventing outbound traffic to our endpoints. If you have a proxy or firewall, make sure the IPs addresses for the DBM endpoints are whitelisted. The IP addresses are contained in the APM block defined at https://ip-ranges.datadoghq.com/.
```
Comment (jmeunier28) [inline content/en/database_monitoring/troubleshooting.md:62]: we can also link to the HA proxy config setup docs? https://docs.datadoghq.com/agent/proxy/?tab=linux

Comment (cswatt) [inline content/en/database_monitoring/troubleshooting.md:62]:
```suggestion
Some common causes of connection failure include proxy setups and firewalls, which outbound traffic to Datadog's endpoints. If you have a proxy or firewall, make sure the IPs addresses for the DBM endpoints are allowed. Refer to the APM block in Datadog's [IP addresses](https://ip-ranges.datadoghq.com/).
```

Comment (EmmaOnThursday) [reply content/en/database_monitoring/troubleshooting.md]: good call, will add

Comment (cswatt) [reply content/en/database_monitoring/troubleshooting.md]: added!

**Review by cswatt — APPROVED** [review]

**Pull request merged**